bridges/primitives/chain-bridge-hub-rococo/src/lib.rs

```diff
@@ -82,3 +82,18 @@ pub const WITH_BRIDGE_ROCOCO_TO_WESTEND_MESSAGES_PALLET_INDEX: u8 = 51;
 
 decl_bridge_finality_runtime_apis!(bridge_hub_rococo);
 decl_bridge_messages_runtime_apis!(bridge_hub_rococo);
+
```

Comment:
I've moved all fee-related constants from `asset-hub` primitives to `bridge-hub` primitives because:
1) now asset hubs are using different different crates to compute total fee (because delivery tx happens at the other chain);
2) tests for those values reside in bridge hubs